components/shared/embedder/lib.rs

```diff
@@ -411,7 +434,7 @@ pub enum EmbedderMsg {
     /// Changes the cursor.
     SetCursor(WebViewId, Cursor),
     /// A favicon was detected
-    NewFavicon(WebViewId, ServoUrl),
+    NewFavicon(WebViewId, Image),
```

Comment:
Hrm. Serializing the image data here isn't ideal.  Can this hold a private IpcSharedMemory internally and expose a slice to the data?

Reply:
It certainly can!